images/virtualization-artifact/pkg/controller/vd/internal/snapshotting.go

```diff
@@ -74,7 +74,7 @@ func (h SnapshottingHandler) Handle(ctx context.Context, vd *virtv2.VirtualDisk)
 			continue
 		}
 
-		resized, _ := conditions.GetCondition(vdcondition.ResizedType, vd.Status.Conditions)
+		resized, _ := conditions.GetCondition(vdcondition.ResizingType, vd.Status.Conditions)
```

Comment:
It should see not a reason, but a status of condition now. 
Add please check for condition observed generation as well.